azure-proto-core/ResourceContainerBase.cs

```diff
@@ -19,16 +19,16 @@ namespace azure_proto_core
     ///     The return type of the Creation methods, this can be Response<typeparamref name="T" /> or a
     ///     long-running response
     /// </typeparam>
-    public abstract class ResourceContainerOperations<TOperations, TResource> : OperationsBase
+    public abstract class ResourceContainerBase<TOperations, TResource> : OperationsBase
```

Comment:
When we change the class name we should change the file name to match

Reply:
The file name is ResourceContainerBase.cs, it has aligned with the class name ResourceContainerBase. Same as rest.